docs/overview.md

```diff
@@ -0,0 +1,100 @@
+# Cumulus overview
+
+This document provides high-level documentation for Cumulus.
+
+## Runtime
+
+Each Substrate blockchain provides a runtime. The runtime is the state transition function of the
+blockchain. Cumulus provides interfaces and extensions to convert a default Substrate runtime into a
+Parachain runtime. Polkadot expects that each runtime exposes an interface for validating a
+Parachain state transition and also provides interfaces to the Parachain to send and receive
+messages of other Parachains.
+
+To convert a Substrate runtime into a Parachain runtime, the following code needs to be added to a
+runtime:
+```rust
+cumulus_runtime::register_validate_block!(Block, Executive);
+```
+
+This macro call expects the `Block` and `Executive` type. It generates the `validate_block` function
+that is expected by Polkadot to validate the state transition.
+
+When compiling a Cumulus augmented runtime, a WASM binary is generated that contains the *full* code
```

Comment:
```suggestion
When compiling a runtime that uses Cumulus, a WASM binary is generated that contains the *full* code
```

What does augmented mean in this context? You mean a runtime which includes cumulus?

Reply:
I see it as Cumulus wraps/extends a runtime. From a developer perspective, you are right and we include Cumulus.